src/containers/HomeLandingPage.tsx

```diff
@@ -46,15 +47,27 @@ const HomeLandingPage = () => {
           tabLabels={tabLabels}
         />
       </div>
-      {events.map((event: EventType) => (
-        <EventCard
-          date={event.eventDate}
-          eventTitle={event.name}
-          address="N/A"
-        />
-      ))}
-      <div className="add-event-container">
-        <AddEventButton />
+      <div className="landing-body">
+        <Grid
+          container
+          direction="row"
+          justify="space-between"
+          alignItems="center"
+          spacing={3}
+        >
+          {events.map((event: EventType) => (
+            <Grid item>
+              <EventCard
+                date={event.eventDate}
+                eventTitle={event.name}
+                address="N/A"
+              />
+            </Grid>
```

Comment:
Getting this console warning `index.js:1 Warning: Each child in a list should have a unique "key" prop.` Please add a `key` prop to `EventCard`. Could be `key={event.name}`.